### DISCUSSION TITLE
Schematics ?

### DISCUSSION THREAD

**Discussion by Maitresinh** [copych/AcidBox#7] [General]
Seems like a nice build, but a basic schematics would help the wiring is different in the 2 pictures)

**Comment by copych** [comment]
@Maitresinh sorry for that. Actually, first pic represents ESP32, while the second shows ESP32-S3. It would be great if you advise some soft to draw the schematics.

**Reply by Maitresinh** [reply]
thanks. I see it know. So there is a capacitor with ESP32 (01) ?
fritzing or even a handraw would work.

**Comment by copych** [comment]
The only reason the capacitor is here is that having it plugged you don't need to push the boot button every time you wanna flash a new firmware.

**Comment by streetuff** [comment]
Maybe this helps. -> https://github.com/copych/AcidBox/discussions/9
